test/index.js

```diff
@@ -303,3 +303,17 @@ test('results in valid syntax when source has trailing comment', (t) => {
   t.doesNotThrow(() => require('./fixtures/trailing-comment'))
   t.end()
 })
+
+test('Set env.DEBUG and captures any calls to `debug` and passes them through pino logger', (t) => {
```

Comment:
can we rephrase the test: something like

"preserves `DEBUG` env independently from `debug` module"